### DISCUSSION TITLE
Search Plugins not working

### DISCUSSION THREAD

**Discussion by Nexus-Broadcast** [LivingWithHippos/unchained-android#252] [Q&A]
New user.  The first day I used this everything worked perfectly.  Today I tried using the search plugins and no results are ever returned from any source.  Ive tried on a couple of different phones.

**Comment by LivingWithHippos** [comment]
Hello, what release are you using, from the play store? I just tried and it works here so:

- use a browser to go the websites manually and check if they can be reached from your country
- delete data and retry

**Comment by Nexus-Broadcast** [comment]
Yes from the play store.  Thank you for replying.  It turns out that
somehow my starlink system was blocking it.  I could reach the plugin
servers on a browser but for some reason the plugins didn't work.
Rebooting starlink made.it work again.

Thank you for a wonderful piece of software.  It saves me a ton of time.

**Comment by Nexus-Broadcast** [comment]
I think I have uncovered a bug.  When downloading single files sometimes it
seems it puts them into a que and if one of the files doesn't complete, it
deleted all the other completed files... So I have been having to download
one file at a time.
